app/actions/loginActions.js

```diff
@@ -0,0 +1,52 @@
+import { push } from 'react-router-redux';
+import { LOGIN_REQUEST, LOGIN_SUCCESS, LOGIN_ERROR } from './actionTypes';
+import loginService from '../services/loginService';
+
+function loginRequest () {
+  return {
+    type: LOGIN_REQUEST,
+    payload: {
```

Comment:
I'll prefer not to use `payload`. Not required, you can of course, but I think you can use directly:

```
return {
     type: LOGIN_REQUEST,
     loggingIn: true
}
```

Reply:
Yes, we can do that.